python/ray/serve/multiplex.py

```diff
@@ -247,7 +246,7 @@ async def unload_model_lru(self) -> None:
             if not inspect.iscoroutinefunction(model.__del__):
                 await asyncio.get_running_loop().run_in_executor(None, model.__del__)
             else:
-                await sync_to_async(model.__del__)()
+                await model.__del__()
```

Comment:
@sihanwang41 I'm reading this correctly and this `sync_to_async` is unnecessary, right? We already check if it's `sync` in the conditional above.

Reply:
yes!